views/login.html

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
If the entire page functioned on one page, then yes. However, since we're right now sending multiple HTML files, you need to define the header on each one. When we have more function HTML working later, we will refactor it to work on one page, and therefore only have one header.